QStreak/View Controllers/Account Setup/AccountSetupViewModel.swift

```diff
@@ -18,6 +18,10 @@ class AccountSetupViewModel {
 
     weak var delegate: AccountSetupViewModelDelegate?
 
+    func getStartedButtonTapped() {
```

Comment:
I don't think we need this?

Reply:
Whoops, I started with something there, but forgot to take it out later. Thanks! I just pushed a revision.